src/validators/movies.validators.ts

```diff
@@ -0,0 +1,39 @@
+import { body, param } from 'express-validator';
+import MovieModel from '../models/movies.model';
+
+export const createMovieValidator = [
+    body('title')
+        .trim()
+        .notEmpty()
+        .withMessage('Please specify a movie title')
+        .custom(value => {
+            return MovieModel.findOne({ title: value }).then(title => {
+                if (title) {
+                    return Promise.reject('Such movie is already exists!');
+                }
+            });
+        }),
+    body('description').trim().isLength({ min: 5 }).withMessage('Min length is 5 characters'),
+    body('releaseDate').trim().toDate(),
+    body('genre').isArray({ min: 1 }).withMessage('Please specify at least one genre'),
+];
+
+export const updateMovieValidator = [
```

Comment:
For improvements, you can combine this logic in two functions in one
Your second function has dublicated code from first

Reply:
Validators has been refactored. I left only one movie/genre validator. MongoId is now checked by a separate validator.